src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1798,6 +1798,12 @@ void Graph::EnforceInferencePrecision() {
             if (node->getOriginalOutputPrecisionAtPort(i) != Precision::FP32)
                 continue;
 
+            // exclude Convert before Range since it may cause precision loss when integter type to LP.
+            // TODO
+            const auto &child = node->getChildEdgesAtPort(i)[0]->getChild();
+            if (child->getType() == Type::Range && node->getType() == Type::Convert)
```

Comment:
This pattern cannot fix issue of gpt-j-6b as there is a reorder inserted in exec graph. We need a more robust pattern or solution.

Reply:
@ceciliapeng2011 sync to the lastest master. The issue will go away.